tests/python/unittest/test_ndarray.py

```diff
@@ -1495,6 +1495,20 @@ def test_dlpack():
             mx.test_utils.assert_almost_equal(a_np, d_np)
             mx.test_utils.assert_almost_equal(a_np, e_np)
 
+@with_seed()
+def test_ndarray_is_inf():
+    data = mx.nd.array([np.inf, -np.inf, np.NINF, -1])
```

Comment:
randomize the shape and size of the array in this test case

Reply:
Thanks for that.